hal/armv7m/imxrt/117x/hal.c

```diff
@@ -88,8 +90,23 @@ addr_t hal_kernelGetAddress(addr_t addr)
 }
 
 
+int hal_memAddMap(addr_t start, addr_t end, u32 attr, u32 id)
+{
+	return mpu_mapAlloc(start, end, attr, id);
+}
+
+
+void hal_memInfo(void)
+{
+	mpu_mapInfo();
+}
+
+
 int hal_cpuJump(addr_t addr)
 {
+	if (mpu_syspageConfig() < 0)
+		return -1;
+
 	syspage_save();
 
```

Comment:
The same case as in imxrt106x.

Reply:
This function will be removed after syspage refactoring.

Reply:
Removed.